.gitignore

```diff
@@ -8,4 +8,5 @@ coverage
 composer.lock
 vendor
 node_modules
-.sass-cache
\ No newline at end of file
+.sass-cache
+.php_cs.cache
```

Comment:
Yes, when you run cs-fixer, you have a cache file that need to be ignored, like here : https://github.com/sonata-project/SonataAdminBundle/blob/master/.gitignore#L10